### DISCUSSION TITLE
Google Analytics 4

### DISCUSSION THREAD

**Discussion by UMP1R3** [blackcowmoo/Grafana-Google-Analytics-DataSource#61] [Q&A]
Hi, does this Grafana plugin works with Google Analytics 4 or just the old Universal Analytics?

Thanks for any answer

**Comment by dllewellyn** [comment]
I'd be quite keen to understand how to set up Ga4? I can only see UA in my drop down atm

**Reply by lcc3108** [reply]
> I'd be quite keen to understand how to set up Ga4? I can only see UA in my drop down atm

@dllewellyn 
Make sure the plug-in version is 0.2.0
The 0.2.0 version has been submitted to Grafana but has not yet been released.

If you are using the 0.2.0 version, please select GA as there are UA and GA on the plugin setting page

**Reply by dllewellyn** [reply]
Aha, thanks I'm using the cloud version so I'm guessing that means I cant do it yet?

**Reply by lcc3108** [reply]
Yes, the cloud version is only available in the official release.

The Grafana Community plugin has a review process by the Grafana team in the release of a new version.

Currently, I submitted a new version, but I am waiting for the review from the Graffana team.

If there is no big problem, I think it will be released soon.

**Comment by lcc3108** [comment]
@UMP1R3 
My answer is too late.

Currently(0.1.5), only UA is supported.
The new plugin version(0.2.0) supports both UAGA4.

The new version is requested to be released and will be released soon if there is no modification after the review by the Graffana team.

**Reply by yititofer** [reply]
Hello, it is still unknown when the new version 0.2.0 will be released through Grafana. I wanted to know if it is possible to obtain real-time users since I need to generate a dashboard with multiple sites and know how many users are in real time in each of them. Is it possible? Maybe it is already possible, but I'm asking just to make sure. Thank you for your work! 
Regards!

**Reply by lcc3108** [reply]
@yititofer 
When I checked, it seems to have been released 3 days ago.

**Reply by lcc3108** [reply]
> I wanted to know if it is possible to obtain real-time users since I need to generate a dashboard with multiple sites and know how many users are in real time in each of them. Is it possible? Maybe it is already possible, but I'm asking just to make sure. Thank you for your work!

Google Analytics plugin is using `runReport api` to load data.
Since there is a `runRealtimeReport api`, I think we should use that api.

I think we need a test.

please request feature at issue
